index.html

```diff
@@ -257,55 +235,61 @@ <h2>The <dfn>PerformanceEntry</dfn> interface</h2>
       };</pre>
       <dl>
         <dt><dfn>name</dfn></dt>
-        <dd>This attribute MUST return an identifier for this
-        <a>PerformanceEntry</a> object. This identifier does not have to be
-        unique.</dd>
+        <dd>
+          This attribute MUST return an identifier for this
+          <a>PerformanceEntry</a> object. This identifier does not have to be
+          unique.
+        </dd>
         <dt><dfn>entryType</dfn></dt>
-        <dd>This attribute MUST return the type of the
-        interface represented by this <a>PerformanceEntry</a> object.
-        <p class="note">Example `entryType` values defined by other
-        specifications include: <code>"mark"</code> and <code>"measure"</code>
-        [[USER-TIMING-2]], <code>"navigation"</code> [[NAVIGATION-TIMING-2]],
-        <code>"resource"</code> [[RESOURCE-TIMING-2]],
-        <!-- TODO: add long task spec reference -->
-         and <code>"longtask"</code>.</p></dd>
+        <dd>
+          This attribute MUST return the type of the interface represented by
+          this <a>PerformanceEntry</a> object.
+          <p class="note">Example `entryType` values defined by other
```

Comment:
should we replace backticks with a code tag here?

Reply:
Here `<a>` too.